debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/transforms/UnwrapFromMongoDbEnvelopeTestIT.java

```diff
@@ -76,6 +77,51 @@ public void afterEach() {
         transformation.close();
     }
 
+    @Test
+    @FixFor("DBZ-563")
+    public void shouldDropTombstoneByDefault() throws InterruptedException {
+        // Use the DB configuration to define the connector's configuration ...
+        config = TestHelper.getConfiguration().edit()
+                .with(MongoDbConnectorConfig.POLL_INTERVAL_MS, 10)
+                .with(MongoDbConnectorConfig.COLLECTION_WHITELIST, "transform.*")
+                .with(MongoDbConnectorConfig.LOGICAL_NAME, "mongo")
+                .build();
+
+        // Set up the replication context for connections ...
+        context = new MongoDbTaskContext(config);
+
+        // Cleanup database
+        TestHelper.cleanDatabase(primary(), DB_NAME);
+
+        // Start the connector ...
+        start(MongoDbConnector.class, config);
+
+        // First insert
+        primary().execute("insert", client -> {
+            client.getDatabase(DB_NAME).getCollection(COLLECTION_NAME)
+                    .insertOne(Document.parse("{'_id': 1, 'dataStr': 'hello', 'dataInt': 123, 'dataLong': 80000000000}"));
+        });
+
+        SourceRecords records = consumeRecordsByTopic(1);
+
+        assertThat(records.recordsForTopic(TOPIC_NAME).size()).isEqualTo(1);
+
+        // Test Delete
+        primary().execute("delete", client -> {
+            client.getDatabase(DB_NAME).getCollection(COLLECTION_NAME).deleteOne(RawBsonDocument.parse("{'_id' : 1}"));
+        });
+
+        records = consumeRecordsByTopic(2);
+        assertThat(records.recordsForTopic(TOPIC_NAME).size()).isEqualTo(2);
+
+        // Test tombstone record is dropped
+        // Note we're getting the second record which is the tombstone
```

Comment:
Ah, this one makes me realize that one last bit for [DBZ-563](https://issues.jboss.org/browse/DBZ-563) is still missing: the unwrap SMT for the relational connectors has [another option](https://debezium.io/docs/configuration/event-flattening/#configuration_options) for dropping deletes, too, via `delete.handling.mode`. Could you perhaps add this here, too, to bring this SMT on par with the other one?

Reply:
Yes I'll start that!
One tiny problem is the default behavior which before was none and now is drop!
We should disclaim that in the release notes